kratos/includes/model_part.h

```diff
@@ -796,6 +796,19 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
      */
     void AddProperties(PropertiesType::Pointer pNewProperties, IndexType ThisIndex = 0);
 
+    /** Returns the Properties corresponding to it's identifier is defined */
+    bool HasProperties(IndexType PropertiesId, IndexType ThisIndex = 0)
+    {
+        auto pprop_it = GetMesh(ThisIndex).Properties().find(PropertiesId);
+        if(pprop_it != GetMesh(ThisIndex).Properties().end()) { //property does exist
+            return true;
+        } else {
+            return false;
+        }
+
+        return false;
+    }
+
```

Comment:
minor, but i would do

    {
        auto pprop_it = GetMesh(ThisIndex).Properties().find(PropertiesId);
        if(pprop_it != GetMesh(ThisIndex).Properties().end()) { //property does exist
            return true;
        } 

        return false;
    }

to save a if statement.